zeppelin-plugins/notebookrepo/mongo/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.zeppelin</groupId>
+        <artifactId>zengine-plugins-parent</artifactId>
+        <version>0.9.0-SNAPSHOT</version>
+        <relativePath>../../../zeppelin-plugins</relativePath>
+    </parent>
+
+    <artifactId>notebookrepo-mongo</artifactId>
+    <packaging>jar</packaging>
+    <version>0.9.0-SNAPSHOT</version>
+
+    <name>Zeppelin: Plugin MongoNotebookRepo</name>
+    <description>NotebookRepo implementation based on MongoDB</description>
+
+    <properties>
+        <plugin.name>NotebookRepo/MongoNotebookRepo</plugin.name>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.mongodb</groupId>
+            <artifactId>mongo-java-driver</artifactId>
+            <version>3.4.1</version>
+        </dependency>
+        <dependency>
+            <groupId>de.flapdoodle.embed</groupId>
+            <artifactId>de.flapdoodle.embed.mongo</artifactId>
```

Comment:
We need to add dependency into [LICENCE](https://github.com/apache/zeppelin/blob/master/LICENSE)  since this is new dependency

Reply:
Is there any document?

Reply:
First check the license of this dependency, then add it to the proper section, e.g. 
![image](https://user-images.githubusercontent.com/164491/63736339-4b54df80-c8b5-11e9-99a2-6ab146423ef5.png)

Reply:
Please first check the license of this dependency, then add it to the proper section, e.g. https://github.com/apache/zeppelin/blob/master/LICENSE#L234